src/main/resources/treeformats.yml

```diff
@@ -51,6 +51,17 @@ formats:
     version:
       regex: "@([^@\\s]+)(\\s\\S+)?$"
       group: 1
+  - format: nuget
+    description: "Nuget tree output"
+    tool: "See https://github.com/Brend-Smits/NugetTree"
+    types:
+      "": "nuget"
+    name:
+      regex: "([^ ]*)"
+      group: 1
+    version:
+      regex: "(?<= )(\\d.+)"
```

Comment:
This appears to match group 1 to anything before the first digit in the line. My suspicion is that this might match "funny" names and end up with corrupted version.

Suggestion is to prefix the version regex with something that makes sure it can only start matching after the name.

Reply:
I do not think this is necessary anymore after changing the Regex from regex: "([^ ]*)" to regex: "(?<= )(\\d.+).
Do you disagree?